sdk/clientcore/core/src/main/java/io/clientcore/core/implementation/instrumentation/fallback/FallbackInstrumentation.java

```diff
@@ -35,10 +42,26 @@ public FallbackInstrumentation(InstrumentationOptions instrumentationOptions,
      * {@inheritDoc}
      */
     @Override
-    public Tracer getTracer() {
+    public Tracer createTracer() {
         return new FallbackTracer(instrumentationOptions, libraryOptions);
     }
 
+    @Override
+    public Meter createMeter() {
+        return NoopMeter.INSTANCE;
```

Comment:
No `FallbackMeter` like we have for `FallbackTracer`? That's fine, but mind adding a comment here why Meter is different than Tracer.